{
    "name": "async-handled-exception",
    "version": "1.0.0"
}
